src/app/user/page.tsx

```diff
@@ -65,22 +58,13 @@ const User = () => {
   const tabFromParams = searchParams?.get('tab') as TabValue
   const defaultTabValue = tabs[tabFromParams]?.value ?? 'holdings'
 
-  const { data: isBuilderOrBacker, isLoading, error } = useIsBuilderOrBacker(address ?? zeroAddress)
-
-  useHandleErrors({
-    error,
-    title: 'Error fetching user data',
-  })
-
-  const showAdditionalContent = !isLoading && isBuilderOrBacker
-
   return (
     <>
       {/* We don't show the tab if it's loading */}
       <Tabs defaultValue={defaultTabValue}>
         {!isConnected && <HeroSection />}
         <JustifyBetweenLayout
-          leftComponent={<UserHeader showAdditionalContent={showAdditionalContent} />}
+          leftComponent={<UserHeader />}
           rightComponent={
             <>
               <SelfContainedNFTBoosterCard />
```

Comment:
The`SelfContainedNFTBoosterCard` is no longer at this level. Remember that it needs to be next to the `BecomeABuilderButton` that is one level above.